__main__.py

```diff
@@ -38,11 +38,12 @@ def login():
         return render_template('login.html', user='', pawd='')
 
 
-@app.route('/logged_in')
-def logged_in():
-    user = request.cookies.get('user', '')
-    pawd = request.cookies.get('pawd', '')
-    return render_template('logged_in.html', user=user, pawd=pawd)

```

Comment:
Could you please check if logged_in.html was removed as we don't need it any more?

Reply:
done